mlflow/server/js/package.json

```diff
@@ -116,16 +121,18 @@
     "jest-canvas-mock": "^2.2.0",
     "jest-localstorage-mock": "^2.3.0",
     "jest-watch-typeahead": "^1.0.0",
-    "prettier": "^2.6.2",
+    "prettier": "^2.8.0",
     "redux-mock-store": "^1.5.3",
-    "typescript": "^4.8.4",
+    "typescript": "^4.9.3",
     "webpack": "^5.69.0"
   },
   "scripts": {
     "//": "IMPORTANT: For compatibility with Windows in MLflow OSS, do NOT inline environment variables in scripts other than `start-web`, `start:mfe` and `test:ci`. Environment variables can instead be set in `mlflow/server/js/scripts/global-setup.js`.",
     "start": "craco start",
     "start:iframe": "HIDE_HEADER=true HIDE_EXPERIMENT_LIST=true SHOW_GDPR_PURGING_MESSAGES=true USE_ABSOLUTE_AJAX_URLS=true SHOULD_REDIRECT_IFRAME=true craco start",
-    "start:dev-proxy": "MLFLOW_DEV_PROXY_MODE=true yarn start:iframe",
+    "start:dev-proxy-fs": "START_FEATURE_STORE=true yarn start:dev-proxy-iframe",
+    "start:dev-proxy-iframe": "MLFLOW_DEV_PROXY_MODE=true yarn start:iframe",
+    "start:dev-proxy": "MLFLOW_DEV_PROXY_MODE=true yarn start:mfe",
```

Comment:
so these commands are non-OSS (also one is FS-specific), however it doesn't seem to impose a big harm - on the other hand, it won't be easy to remove it from here without some specific customly made copybara commands

Reply:
Yeah, I think it's fine to keep these commands unless they are harmful or contain confidential information.